tests/PySys/plugin_apama/apama_plugin_install/run.py

```diff
@@ -0,0 +1,24 @@
+import sys
+
+from environment_apama_plugin import ApamaPlugin
+
+"""
+Validate apama plugin install command.
+Using the apama project zip file in this test's Input directory as the test artefact.
+This project deploys a mon file named "TedgeDemoMonitor" into the apama correlator.
+"""
+
+
+class ApamaPluginInstallTest(ApamaPlugin):
+
+    def setup(self):
+        super().setup()
+        # Assert that an apama project is not installed on the machine before test
+        self.assert_project_installed(negate=True)
+
+    def execute(self):
+        self.plugin_install(self.input + "/quickstart.zip")
```

Comment:
I think using a wrapper is not a good idea, because this somehow hides the command under test.

I would prefer an explicit call:
```suggestion
        self.startProcess(
            command=self.sudo,
            arguments=["/etc/tedge/sm-plugins/apama", "install",
                       "project", "--file", project_archive_path],
            stdouterr="plugin_install"
        )
```